src/util.rs

```diff
@@ -0,0 +1,93 @@
+use std::mem::transmute;
+
+use ff::PrimeField;
+use ruint::aliases::U256;
+
+/// Assumes little endian
+pub fn u256_as_limbs(uint: U256) -> [u32; 8] {
+    let limbs = *uint.as_limbs();
+    unsafe { transmute(limbs) }
+}
+
+/// Assumes little endian
+pub fn limbs_as_u256(limbs: [u32; 8]) -> U256 {
+    let limbs: [u64; 4] = unsafe { transmute(limbs) };
+    U256::from_limbs(limbs)
+}
+
+/// Assumes little endian
+pub fn ff_as_limbs<F: PrimeField>(f: F) -> [u32; 8] {
+    let binding = f.to_repr();
+    let repr: [u8; 32] = binding.as_ref().try_into().unwrap();
```

Comment:
Better error definition and propagation could be defined here. Or at least an expect giving us some context :)